server-templates/typeInfoAnnotation.mustache

```diff
@@ -0,0 +1,29 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Please use the same header style as in other `.mustache` files (e.g. `server-templates/queryParams.mustache`)